requirements-dev.txt

```diff
@@ -1,6 +1,6 @@
 autopep8>=1.3.5,<1.4.0
 Babel
 flake8>=3.5.0<3.6.0
-isort>=4.2.2,<4.3.0
+isort>=4.3.4,<4.4.0
 pycodestyle>=2.3.1,<2.4.0
-sentry-flake8>=0.0.1
+sentry-flake8>=0.0.1
```

Comment:
We lost our linebreak here !

Reply:
GitHub online editor for the lose. I'll put the newline back.